app/controllers/addons_controller.rb

```diff
@@ -0,0 +1,51 @@
+class AddonsController < ApplicationController
+  def index
+    @addons = current_employee.restaurant.addons
+  end
+
+  def show
+    @addon = Addon.find(params[:id])
+  end
+
+  def new
+    @addon = Addon.new
+  end
+
+  def create
+    @addon = Addon.new(addon_params)
+    if @addon.save
+      flash[:notice] = "Created Addon!"
+      redirect_to addons_path
+    else
+      render :new, status: :unprocessable_entity

```

Comment:
Add Flash.now[:alert] here if the object is not saved.

Reply:
Done.